src/frontend/components/SearchAndFilter/SearchAndFilter.js

```diff
@@ -0,0 +1,155 @@
+import React, { useState, useRef } from "react";
+import { useOnClickOutside } from "../../hooks";
+import { useFilters, useNotes } from "../../contexts";
+import "./SearchAndFilter.css";
+import {
+  FILTER_PRIORITIES,
+  FILTER_SORT_BY,
+  FILTER_TAGS,
+} from "../../constants";
+
+export const SearchAndFilter = () => {
+  const filtersRef = useRef();
+  const [showFilters, setShowFilters] = useState(false);
+
+  useOnClickOutside(filtersRef, () => setShowFilters(false));
+
+  const { uniqueTags } = useNotes();
+
+  const { filters, dispatchFilters } = useFilters();
+  const { sortBy, priorities, tags } = filters;
+
+  return (
+    <div className="search-and-filter">
+      <span className="material-icons search-icon">search</span>
+      <input
+        className="search-input"
+        type="text"
```

Comment:
you can add type : search it will be more symantic

Reply:
Thank you!